core/validation/const.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+import pandas as pd
+
+# values we treat as null
+NA_VALUES = {"", "< Select >", np.nan, None, pd.NA}
```

Comment:
Do we actually want `< Select >` here?

Reply:
It was written into the tests for leading_factor_of_delay. Looking at the extraction function for project progress, we replace all instances with an empty string. I've now removed all mention of < Select > from the validation